src/main.rs

```diff
@@ -1,9 +1,10 @@
-use std::{env,path,process};
-
+#[macro_use]
+extern crate nix;
 extern crate byteorder;
 extern crate getopts;
 extern crate libudev;
-#[macro_use] extern crate nix;
+
+use std::{env,path,process};
```

Comment:
Not quite sure about this, it violates my usual order of imports which is standard library, external libraries then internal modules.